src/library_html5.js

```diff
@@ -32,6 +36,14 @@ var LibraryJSEvents = {
     // Track in this field whether we have yet registered that __ATEXIT__ handler.
     removeEventListenersRegistered: false, 
 
+#if HTML5_SUPPORT_DEFERRING_USER_SENSITIVE_REQUESTS
+    // If positive, we are currently executing in a JS event handler.
+    inEventHandler: 0,
+    // If we are in an event handler, specifies the event handler object from the eventHandlers array that is currently running.
+    currentEventHandler: null,
+#endif
+*/
```

Comment:
maybe a comment here to clarify where the end of the comment is for?

Reply:
There is already a comment about that on lines 10-12, which spans the whole /* */ block.

Reply:
True, but it's not that easy to see where it ends. Although I guess that's just in the diff here - locally with proper syntax highlighting I would see it ok...